src/main/kotlin/baseball/controller/GameController.kt

```diff
@@ -0,0 +1,29 @@
+package baseball.controller
+
+import baseball.domain.*
+import baseball.view.View
+
+class GameController(val view: View, val numbersGenerator: NumbersGenerator = RandomNumbersGenerator()) {
+
+    fun startGame() {
+        view.printStartGameMessage();
+        do {
+            playEachGame()
+        } while (isRetryGame())
+    }
+
+    private fun playEachGame() {
+        val game = Game(numbersGenerator)
+        do {
+            val userNumbers = Numbers.from(view.inputUserNumbers())
+            val roundResult = game.playRound(userNumbers)
+            view.printRoundResult(roundResult)
+        } while (!roundResult.isWin())
+        view.printWinMessage()
+    }
+
+    private fun isRetryGame(): Boolean {
+        val retryCommand = RetryCommand.from(view.inputRetryCommand())
+        return retryCommand.equals(RetryCommand.RETRY);
```

Comment:
코틀린에서는 `==`가 `equals`와 같이 동등성을 비교한다고 하네요! `==`가 내부적으로 equals를 사용한대요

```suggestion
        return retryCommand == RetryCommand.RETRY
```

Reply:
으앗차ㅏ차 감사합니다